### ISSUE TITLE
1.4.11: please list color values with examples

### ISSUE THREAD

**Issue by fstorr** [w3c/wcag#917]
It would be really useful if colors could be listed next to their examples. Because the examples are images rather than "live" code, checking them to help understand the document needs a color picker. Color pickers are at the mercy of properly-calibrated screens, and depending where I'm working (I work in different places with different screens), I get different results. 

For example, the description for Figure 4A states that the contrast ratio is 5.6:1, but I'm currently getting a result of 5.99:1 using two different contrast-checking tools. Having the colors listed would also help clarify whether the borders of radio buttons in Figure 5 have, or don't have, a 3:1 contrast. I'm seeing 2.55:1 at the moment and don't know if that's accurate.

Thanks

**Comment by Myndex** [comment]
Hi @fstorr

If you're using Digital Color Meter to sample screen pixels on your monitor, that takes the monitor profile to account. As a result, you may get different values and were actually used in the CSS. 

If the example you're looking at is a live webpage graphic, that all you need to do is highlight the element, and under development tools in your browser inspect that element to see the CSS values actually used. 

If on the other hand the example element is a JPEG or a PNG, you can load that into Gimp or Photoshop which will allow you to sample colors using the colorspace that the image was created with (which should be sRGB).

When sampling colors on your monitor, you can temporarily set the monitor profile to a generic sRGB profile, and that should allow you to use Digital Color Meter to get correct values.

Some other color meters, like Classic Color Meter in the Mac app store, have additional settings that can help provide additional color information.

**Comment by fstorr** [comment]
@Myndex 

Thanks for the reply. I know how to audit colors and how to use dev tools / web inspectors to get CSS values. I also know that using a Digital Color Meter isn't accurate because it relies on monitor calibration—that's the crux of my problem and why I'm suggesting clarifying the Understanding document. What I'm asking for is to add the color values to any color-related example in an Understanding document.

For example, the radio-button example in 1.4.11 would be much more helpful if the the ```<figcaption>``` element said something similar to: "A series of radio buttons each with a border color of ```#a2a2a1``` [if that's the actual color]. The first radio button shows the default state, the second (completely filled with a color of ```#797979```),  and third (completely filled with the same color at the border) show the radio button selected and filled with a color that contrasts with the color adjacent to the component. The fourth example (partially filled with a color of ```#797979```) shows the state indicator contrasting with the component colors."

Side note: there's a typo—"contasts"—in the current Understanding document.

**Comment by alastc** [comment]
I'm in-progress with an update to the understanding doc, I'll add the values.

**Comment by alastc** [comment]
The understanding has been updated (and will be published soon) including the hex values. Closing based on the changes in #931
